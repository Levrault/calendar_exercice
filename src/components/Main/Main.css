.main {
  display: flex;
  justify-content: center;
  margin-top: $space-l;
}

