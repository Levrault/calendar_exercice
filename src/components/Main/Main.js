import React from 'react';
import PropTypes from 'prop-types';
import styles from './Main.css';

/**
 * Main div container
 * @param {children} children
 */
const Main = ({ children }) => (
  <main className={styles.main}>
    {children}
  </main>
);

Main.proptypes = {
  children: PropTypes.node.isRequired
};

export default Main;
