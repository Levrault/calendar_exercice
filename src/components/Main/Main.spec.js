import React from 'react';
import renderer from 'react-test-renderer';
import Main from './Main';

describe('Main component', () => {
  it('should render without problem', () => {
    const component = renderer.create(
      <Main><div>Main</div></Main>,
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
