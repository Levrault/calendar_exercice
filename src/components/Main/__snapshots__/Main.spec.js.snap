// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main component should render without problem 1`] = `
<main>
  <div>
    Main
  </div>
</main>
`;
